app/views/_footer.html.erb

```diff
@@ -0,0 +1,5 @@
+<footer id="footer" class="navbar navbar-inverse">
```

Comment:
as of https://github.com/projecthydra/sufia/pull/2166 this should be in `app/views/shared/_footer.html.erb`